OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+
+# A sample command will look ike this -
+
+docker run -it --name WCCAdminContainer --network=WCContentNET -p 7001:7001 -v /scratch/wccdocker/volumes/wccontent:/u01/oracle/user_projects --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/createDomainandStartAdmin.sh; /bin/bash"
+```
+**Note:** The above command deletes any previous RCU with the same prefix if **DB_DROP_AND_CREATE=true**
+
+The docker run command creates the container as well as starts the Admin Server in sequence given below:
+
+* Node Manager
+* Admin Server
+
+When the command is run for the first time, we need to create the domain and configure the content managed servers, so following are done in sequence:
+
+* Loading WebCenter Content schemas into the database
+* Creating WebCenter Content domain
+* Extending WebCenter Content domain for associated products (e.g. IPM, ADFUI etc.) - based on **component** env varibale. 
+* Configuring Node Manager
+* Starting Node Manager
+* Starting Admin Server
+
+#### B. Stopping Admin Container
+```
+docker container stop WCCAdminContainer
+```
+
+#### C. Starting Admin Container
+```
+docker container start -i WCCAdminContainer
+```
+
+### 5.1.3. Managed Server Container (WCContentContainer)
+
+#### A. Creating and Running WCContent Container
+Run the following command to create the WCContent Managed Server container:
+
+```
+docker run -it --name WCContentContainer --network=WCContentNET -p <UCM_HOST_PORT>:<UCM_PORT> -p <IBR_HOST_PORT>:<IBR_PORT> -p <UCM_INTRADOC_PORT>:<UCM_INTRADOC_PORT> -p <IBR_INTRADOC_PORT>:<IBR_INTRADOC_PORT> --volumes-from WCCAdminContainer --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0  /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterContent.sh; /bin/bash"
+
+# A sample command will look like this -
+
+docker run -it --name WCContentContainer --network=WCContentNET -p 16200:16200 -p 16250:16250 -p 4444:4444 -p 5555:5555 --volumes-from WCCAdminContainer --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0  /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterContent.sh; /bin/bash"
+```
+The docker run command creates the container as well as starts the WebCenter Content managed servers. 
+
+Note: 1. If manged servers need to be accessed through host ports different from container ports, then intended host port values needs to be supplied as part of -p option of the 'docker run'
+command mentoned above (for ex. -p 16201:16200 and -p 16251:16250). The same port value needs to be updated in the webcenter.env.list as UCM_HOST_PORT and IBR_HOST_PORT.
```

Comment:
Please wrap `webcenter.env.list` and `UCM_HOST_PORT` in back ticks to indicate they are literals.